UITests/AuthenticationTests.swift

```diff
@@ -70,13 +70,15 @@ class AuthenticationTests: KIFTestCase {
 
     fileprivate func loadAuthPage() {
         EarlGrey.select(elementWithMatcher: grey_accessibilityID("url")).perform(grey_tap())
-        EarlGrey.select(elementWithMatcher: grey_accessibilityID("address")).perform(grey_typeText("\(webRoot!)/auth.html\n"))
+        EarlGrey.select(elementWithMatcher: grey_accessibilityID("address")).perform(grey_replaceText("\(webRoot!)/auth.html"))
+        EarlGrey.select(elementWithMatcher: grey_accessibilityID("address")).perform(grey_typeText("\n"))
```

Comment:
ok, and I now see it is a replaceText first, then a typeText, I didn't notice that before